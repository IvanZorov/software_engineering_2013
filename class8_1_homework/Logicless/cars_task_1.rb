#/usr/bin/ruby

require 'csv'

bmw = 0
ren = 0
vol = 0
n = 0

CSV.foreach("cars.csv", "r") do |row|	
b = row[1] 
a = ARGV[0]
 
if b == a
    n += 1
end

if b == "Bmw"
    bmw += 1
end

if b =="Renault"
    ren += 1
end

if b =="Volkswagen"
    vol += 1 
end

info = File.open("cars_info.txt", "w")
info.puts "BMW = ", bmw,"Renault = ", ren,"Volkswagen = " ,vol,"Vuvedena marka " ,a,"broi avtomobili " ,n
info.close

end
