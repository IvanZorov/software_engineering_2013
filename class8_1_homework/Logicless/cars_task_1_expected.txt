BMW = 
19
Renault = 
18
Volkswagen = 
27
Vuvedena marka 
Opel
broi avtomobili 
12
